docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,167 @@
+- Feature flags
+- Expanding existing APIs
+
+In all cases, experimental code should be treated as unstable and can change
+between release versions for the package that it is being imported from.
+
+- The API is not fixed, and is likely to change
+- The API is not bound by semver
+- The component, export, feature, etc. may change, be renamed, or removed in the
+  future without warning
+
+## Components and exports prefixed with `UNSTABLE_`
+
+For experimental or unstable exports, we use the `UNSTABLE_` prefix. For
+example:
+
+```js
+// An unstable method
+function UNSTABLE_layout() {
+  // ...
+}
+
+// An unstable variable
+const UNSTABLE_meta = {
+  // ...
+};
+
+// An unstable component will retain its name, specifically for things like
+// the rules of hooks plugin which depend on the correct casing of the name
+function ExampleComponent(props) {
+  // ...
+}
+
+// However, when we export the component we will export it with the `UNSTABLE_`
+// prefix. (Similar to React.unstable_Suspense, React.unstable_Profiler)
+// however we will use it in all caps format
+export { default as UNSTABLE_ExampleComponent } from './components/ExampleComponent';
+```
+
+For teams using these features, they will need to import the functionality by
+using the `UNSTABLE_` prefix. For example:
+
+```jsx
+import { UNSTABLE_ExampleComponent as ExampleComponent } from '@lmc-eu/spirit-web-react';
+```
+
+### Documenting components and exports prefixed with `UNSTABLE_`
+
+Within the experimental section of the `@lmc-eu/spirit-web-react` Storybook, there are
+individual entries for each unstable export. These contain stories and written
+documentation outlining the functionality of these components and exports.
+
+## Feature flags
+
+Some Spirit packages ship with available feature flags. These feature flags
+enable new behavior and styling, allowing you to opt-in to new breaking changes
+while remaining on the current major version. When a new feature flag is
+introduced it is marked `false` or "off" by default to ensure backwards
+compatibility. A feature flag may be configured in JavaScript, SASS, or both.
+
+All the currently available feature flags, as well as how to enable them, is
+documented in the each package main README file in the section `Feature Flags`.
+
+### Feature flag naming convention
+
+All feature flags follow a prefix naming convention that indicate status.
+
+#### Flags prefixed with `enable-*`
+
+- Contain new features that we'd like consuming projects to test
+- Are generally stable and unlikely to change but may change based on user
+  feedback
+- May require some manual migration or code changes within your project
+- Are documented in the main package README file
+- Need user feedback to ensure we've met all concerns relating to this feature
+
+If you use these flags, make sure to check our release notes where we'll outline
+any changes to them across our releases.
+
+#### Flags prefixed with `enable-v#-*`
```

Comment:
as it is now documented, it differs from how we used it. Pre Release 1 🍍, we used enable-v2-alert-link, meaning we introduced this flag and will remove it in v2. 

TBH I find this section confusing and would maybe avoid it at all. In the current release 🍌 we didn't use the `-vX-` infix and I think we can stick with it. IMHO most of the time we will remove all Feature flags during the release process and won't keep any. So we don't need to "commit" to future major releases and simplify the usage. Another argument against the version infix in the name is the mismatch of package versions. For example if I would have introduce FF in web package in current version `1.x.x` and use it in twig as classname, but twig is in version `2.x.x` is it weird. You would end up with this:

```twig
<div class="spirit-enable-v2-rainbow-alert"> // web v1 classname with commit to remove it v2
 <Alert color="🌈" /> // twig v2 component
</div>
```

Reply:
I have changed the copy. I think it should be much clear now that we use version infix only for marking the feature flags that will be turned on by default in the next major release. E.g. we will remove them and the feature will be the default.